website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+* `annotate` - (Optional) Add this text as a note on the resulting incident.
+* `pagerduty_automation_actions` - (Optional) Configure a [Process Automation](https://support.pagerduty.com/docs/event-orchestration#process-automation) associated with the resulting incident.
+  * `action_id` - (Required) Id of the Process Automation action to be triggered.
+* `automation_actions` - (Optional) Create a [Webhook](https://support.pagerduty.com/docs/event-orchestration#webhooks) associated with the resulting incident.
+  * `name` - (Required) Name of this Webhook.
+  * `url` - (Required) The API endpoint where PagerDuty's servers will send the webhook request.
+  * `auto_send` - (Optional) When true, PagerDuty's servers will automatically send this webhook request as soon as the resulting incident is created. When false, your incident responder will be able to manually trigger the Webhook via the PagerDuty website and mobile app.
+  * `headers` - (Optional) Specify custom key/value pairs that'll be sent with the webhook request as request headers.
+    * `key` - (Required) Name to identify the header
+    * `value` - (Required) Value of this header
+  * `parameters` - (Optional) Specify custom key/value pairs that'll be included in the webhook request's JSON payload.
+    * `key` - (Required) Name to identify the parameter
+    * `value` - (Required) Value of this parameter
+* `severity` - (Optional) sets Severity of the resulting alert. Allowed values are: `info`, `error`, `warning`, `critical`
+* `event_action` - (Optional) sets whether the resulting alert status is trigger or resolve. Allowed values are: `trigger`, `resolve`
+* `variables` - (Optional) Populate variables from event payloads and use those variables in other event actions.
+  * `name` - (Required) The name of the variable
+  * `path` - (Required) Path to a field in an event, in dot-notation. This supports both PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) and non-CEF fields. Eg: Use `event.summary` for the `summary` CEF field. Use raw_event.fieldname to read from the original event `fieldname` data.
+  * `type` - (Required) Only `regex` is supported
+  * `value` - (Required) The Regex expression to match against. Must use valid [RE2 regular expression](https://github.com/google/re2/wiki/Syntax) syntax.
+* `extractions` - (Optional) Replace any CEF field or Custom Details object field using custom variables.
+  * `template` - (Optional) A value that will be used to populate the `target` field. The configuration can include variables extracted from the payload by using string interpolation. Eg: If you have defined a variable called `hostname` you can set extraction `template` to `High CPU on variables.hostname server` to use the variable in extraction.  This field can be ignored for `regex` based replacements.
```

Comment:
**Blocker:** Missing curly braces within the template. In this case, I think re-using the wording from the UI's help modal might be nice!

![Event Orchestration 2022-05-20 15-14-37](https://user-images.githubusercontent.com/1933/169619168-5c8f950d-2348-4844-87b8-93a8ffba2a66.png)


```suggestion
  * `template` - (Optional) A string that will be used to populate the `target` field. You can reference variables or event data within your template using double curly braces. For example:
     <br />Use variables named `ip` and `subnet` with a template like: `{{variables.ip}}/{{variables.subnet}}`
     <br />Combine the event severity & summary with template like: `{{event.severity}}: {{event.summary}}`
```